src/snowflake/snowpark/udf.py

```diff
@@ -63,6 +66,12 @@ def __init__(
         self._input_types = input_types
         self._is_return_nullable = is_return_nullable
 
+        api_call_source = api_call_source or "UserDefinedFunction.__init__"
+        session = session or snowflake.snowpark.session._get_active_session()
+        session._conn._telemetry_client.send_function_usage_telemetry(
+            api_call_source, TelemetryField.FUNC_CAT_CREATE.value
```

Comment:
Similarly, we should add telemetry to the actual creation of UDF.

Reply:
is `__call__` the right place to send the usage telemetry API?

Reply:
yes